### PR TITLE
created two_sum function and tests

### PR DESCRIPTION
**Pull request by AmberRose2**
here is the two_sum function 
created tests
passed tests

### CODE REVIEW

File: functions/two_sum.py [hunk 1]
```diff
@@ -1,2 +1,5 @@
 def twoSum(n, t):
-    pass
```
Comment (J-J-D) [inline functions/two_sum.py:2]: Great job deleting pass

File: tests/two_sum_test.py [hunk 2]
```diff
@@ -1,2 +1,39 @@
...
+def test_returns_0_1_for_3_3_with_target_6():
+    # Arrange
+    n = [3, 3]
+    t = 6
+
+    # Act
+    answer = twoSum(n, t)
+
+    # Assert
+    assert answer == [0, 1]
```
Comment (J-J-D) [inline tests/two_sum_test.py:39]: You came up with this code wicked fast

**Review by J-J-D — APPROVED** [review]
Youre kind, smart and funny.

File: tests/two_sum_test.py (hunk 2 above)
Comment (AmberRose2) [reply tests/two_sum_test.py:39]: thanks

**Comment by J-J-D** [comment]
We rock

**Pull request merged**